Reportedly/Views/Toast/ToastPresenter.swift

```diff
@@ -40,7 +40,7 @@ final class ToastPresenter {
         notificationView?.setTitle(message)
         
         notificationView?.snp.makeConstraints {
-            $0.width.equalToSuperview().offset(-16)
+            $0.width.equalToSuperview().offset(-CGFloat.spacer4)
```

Comment:
I think we can use `inset` instead of `offset` to use positive value. What do you think? 😅

Reply:
yeah sure, let me apply and test it, back then I tried to use inset but seems it doesn't work properly since for this case the view goes out of screen so 😅